subsys/debug/Kconfig

```diff
@@ -164,6 +164,13 @@ config ASSERT_NO_FILE_INFO
 	  in which the assertion occurred. Enabling this will save
 	  target code space, and thus may be necessary for tiny targets.
 
+config ASSERT_NO_COND_INFO
+	bool "Disable condition info for asserts"
+	help
+	  This option removes condition expression of the assertion that
```

Comment:
This sounds a bit unclear to me. Can you reword to something along the lines of "Remove condition expression from the printed assertion message...". Or "Don't print condition expression in the assertion message..."

Reply:
Will improve description.